dio/lib/src/dio_event_processor.dart

```diff
@@ -41,41 +32,20 @@ class DioEventProcessor implements EventProcessor {
       contexts: contexts,
     );
 
-    final innerDioStackTrace = dioError.stackTrace;
-    final innerDioErrorException = dioError.error as Object?;
-
     // If the inner errors stacktrace is null,
-    // there's nothing to create chained exception
-    if (innerDioStackTrace == null) {
+    // there is no chained exception
+    if (dioError.stackTrace == null) {
       return event;
     }
 
-    try {
-      final innerException = _sentryExceptionFactory.getSentryException(
-        innerDioErrorException ?? 'DioError inner stacktrace',
-        stackTrace: innerDioStackTrace,
-      );
-
-      final exceptions = _removeDioErrorStackTraceFromValue(
-        List<SentryException>.from(event.exceptions ?? <SentryException>[]),
-        dioError,
-      );
-
-      return event.copyWith(
-        exceptions: [
-          innerException,
-          ...exceptions,
-        ],
-      );
-    } catch (e, stackTrace) {
-      _options.logger(
-        SentryLevel.debug,
-        'Could not convert DioError to SentryException',
-        exception: e,
-        stackTrace: stackTrace,
-      );
-    }
-    return event;
+    final exceptions = _removeDioErrorStackTraceFromValue(
+      List<SentryException>.from(event.exceptions ?? <SentryException>[]),
+      dioError,
+    );
+
+    return event.copyWith(
+      exceptions: exceptions.reversed.toList(), // Inner before DioError
+    );
```

Comment:
Is this just because of the code above? If not, this should probably be done in the Sentry SDK.

Reply:
Yeah, we were inserting the inner exception before the DioError exception. Did this to have the same behaviour as before.